cachito/web/models.py

```diff
@@ -674,7 +674,7 @@ def _validate_gitsubmodule_exclusivity(pkg_manager_paths, mutually_exclusive):
         if any(path != "." for path in paths):
             msg = (
                 f"Cannot process non-root packages with '{pkg_manager}' "
-                "when 'git-submodule' is also set"
+                "when 'git-submodule' is also set. Consider using multiple upstream repos."
```

Comment:
Ideally, the error message should mention how to solve the problem. Here, the solution would typically be to remove the git submodules and instead process the upstream repos (corresponding to the submodules) directly.

Also, since processing multiple upstream repos is not a Cachito feature, the message may need to say that the individual repos should be processed in separate requests.